src/ui/map.js

```diff
@@ -2149,7 +2164,7 @@ class Map extends Camera {
      * @returns {Map} this
      * @private
      */
-    _render() {
+    _render(paintStartTimeStamp: number) {
```

Comment:
Could you update this new parameter from the docs?

Reply:
done in 15a13b57d19e2d84f002d8ad17efb2b6612cdab5